test/e2e/components/vl-map-action.js

```diff
@@ -1,12 +1,15 @@
 const {VlElement} = require('vl-ui-core').Test;
+const VlMap = require('./vl-map');
 
 class VlMapAction extends VlElement {
   async isActive() {
     return this.hasAttribute('active');
   }
 
   async getMap() {
-    return new VlElement(this.driver, await this.driver.executeScript('return arguments[0]._mapElement', this));
+    const map = await new VlMap(this.driver, await this.driver.executeScript('return arguments[0]._mapElement', this));
+    await this.driver.executeScript('arguments[0].ready', map);
+    return map;
   }
```

Comment:
Vermoedelijk niet meer nodig door refactor van vl-map e2e page object.

Reply:
das te zien of ge eerst dan op de page de map gaat vragen of eerst de action, dus just to be sure zou ik het hier ook doen nee?

Reply:
Je hebt gelijk. het is inderdaad veiliger om het hier ook toe te passen vooral dan voor afnemers van de e2e component page objecten.